internal/daemon/vendor-specific-plugins/marvell/main.go

```diff
@@ -398,6 +398,9 @@ func getInterfaceName(deviceID string) (string, error) {
 // enableIPV6LinkLocal function to enable the IPv6 Link Local Address on the given Interface Name
 // It will return the error
 func enableIPV6LinkLocal(interfaceName string, ipv6Addr string) error {
+	// Tell NetworkManager to not manage our interface.
```

Comment:
Do we need nsenter? Shouldn't this be a host network pod instead?

Reply:
hm. Maybe.

for one, currently the VSP container doesn't install /usr/bin/nmcli. But doing so might also not be desirable, because the NetworkManager version on the host might not be compatible with the nmcli version inside the container (in practice, it likely would work). We could instead mount parts of /usr/bin and /usr/lib (or all of /) inside the container, and try to use the host's nmcli inside the container. That sounds also cumbersome, to load the proper shared libraries (maybe a chroot would work).

Also, for nmcli to talk to D-Bus, we need to make `/var/run/dbus/system_bus_socket` accessible.

This probably could also be done.

But is there a problem with using nsenter?

Reply:
Does it make sense to use netlink library instead? What's cost/benefit trade-off?